scripts/cypress_run.py

```diff
@@ -44,7 +44,9 @@ def run_cypress_for_test_file(
     os.environ["TERM"] = "xterm"
     os.environ["ELECTRON_DISABLE_GPU"] = "true"
     build_id = generate_build_id()
-    browser = os.getenv("CYPRESS_BROWSER", "chrome")
+    browser = os.getenv("CYPRESS_BROWSER", "electron")
```

Comment:
a bit concerned to bring yet another engine in the mix, specifically for tests/automation. Supporting two is already a challenge, and thinking now that we should just pick one engine, either Chrome or Firefox, and use is consistently for thumbnail-generation, alert & reports and in e2e tests

Reply:
Also confused about what electron `--headless` means, isn't electron a "head" by definition?  Did a tiny bit of research just now, and Electron uses Blink (a fork of webkit), Chrome/Chromium use Blink as well, and FF uses Gecko. So chrome/chromium/electron in headless more should be pretty much the same AFAIK